tests/testthat/test-post_bug.R

```diff
@@ -0,0 +1,7 @@
+cli::test_that_cli(configs = "plain", "post_bug() works", {
+    skip_if_not(interactive())
+    url <- paste0(missing_host(), "rest/bug")
```

Comment:
Great approach to build up the url, but the function is incomplete and still we need to think what will return to the user. That will depend on what returns the API. If possible it would be nice to return the ID of the bug to be able to look it up on the browser. (but don't test that on this PR) 
Finally you added mockery to mock post_bug I don't see any mock function of post_bug being tested.